extensions/big-data-cluster/src/bigDataCluster/dialog/bdcDashboardOverviewPage.ts

```diff
@@ -367,6 +367,7 @@ function createServiceEndpointRow(modelBuilder: azdata.ModelBuilder, container:
 	copyValueCell.iconPath = IconPathHelper.copy;
 	copyValueCell.onDidClick(() => {
 		vscode.env.clipboard.writeText(endpoint.endpoint);
+		vscode.window.showInformationMessage(localize('copiedEndpoint', "Endpoint '{0}' copied to clipboard", getEndpointDisplayText(endpoint.name, endpoint.description)));
```

Comment:
Just confirming since this is in your commit message, but not explicitly occurring at this level, but `showInformationMessage` always triggers an aria alert?

Reply:
Yup. 

https://github.com/microsoft/azuredatastudio/blob/master/src/vs/workbench/browser/parts/notifications/notificationsAlerts.ts#L33